docs/spec/crosschain/messages.md

```diff
@@ -87,28 +87,47 @@ message MsgNonceVoter {
 Casts a vote on an outbound transaction observed on a connected chain (after
 it has been broadcasted to and finalized on a connected chain). If this is
 the first vote, a new ballot is created. When a threshold of votes is
-reached, the ballot is finalized. When a ballot is finalized, if the amount
-of zeta minted does not match the outbound transaction amount an error is
-thrown. If the amounts match, the outbound transaction hash and the "last
-updated" timestamp are updated.
+reached, the ballot is finalized. When a ballot is finalized, the outbound
+transaction is processed.
 
-The transaction is proceeded to be finalized:
-
-If the observation was successful, the status is changed from "pending
-revert/outbound" to "reverted/mined". The difference between zeta burned
+If the observation was successful, the difference between zeta burned
```

Comment:
```suggestion
If the observation is successful, the difference between zeta burned
```
Shouldn't we use the present? Same comment for below

Reply:
ddd542b0

Reply:
ddd542b0